socorro/schemas/socorro-data-1-0-0.schema.yaml

```diff
@@ -18,6 +18,7 @@ definitions:
 
   simple_types:
     enum:
+      - "any"
```

Comment:
I added `any` so that I could include `metadata` in the schema. This is a collection of collector-generated fields that capture what the payload looked like, the throttling rule that accepted the crash report, etc. These fields are diagnostic and relevant only to socorro engineers debugging collector issues. It's helpful to be able to make changes to the collector without having to do a lock-step set of changes with the rest of socorro across multiple repositories. That's what went into making `metadata` type `any` and using `pattern_properties` to cover all the sub fields.